src/components/LoginButton/LoginButton.style.ts

```diff
@@ -0,0 +1,26 @@
+import Theme from '@/styles/Theme';
+import styled from '@emotion/styled';
+
+const LoginButtonStyled = styled.div`
```

Comment:
p5;
큰 차이는 없을 것 같은데 의미적인 태그를 부여하기 위해 button태그로 만드는 건 어떨까요?
```suggestion
const LoginButtonStyled = styled.button`
```